Server/World.cs

```diff
@@ -288,7 +288,7 @@ public DataEntry(SaveData data, int typeID, string typeName, long pos, int lengt
 					{
 						Console.WriteLine("Error: Type '{0}' was not found. Delete all of those types? (y/n)", typeName);
 
-						if (Console.ReadKey(true).Key == ConsoleKey.Y)
+						if (Insensitive.Equals(Console.ReadLine(), "Y"))

```

Comment:
Again, no. Service and NoConsole have two different intended purposes. Please see my other replies.